src/main/modules/Validation.js

```diff
@@ -0,0 +1,66 @@
+export class Validation {
```

Comment:
Hi, do you have test for this new class?

Reply:
I think they don't

Reply:
No, we don't have test for this class, because it's already covered by Execution.test.js, We noticed that when we were developing the unit test, you can check the coverage as well, but that's a good observation.

Reply:
![image](https://user-images.githubusercontent.com/87339632/129209867-1679cfcc-69f1-48a8-b9fd-26160370051a.png)

I attach the actual coverage, i forgot that you can not check that before the merge, I'm sorry. :D

Reply:
ok, then I'll approve it. Great job

Reply:
Are you sure it would be redundant? Because even if the implementation is similar is still another class so to keep best practice is better to make the test and if looks redundant try to do something like error handling with the test or something like that, but i think it's still necessary to do the tests

Reply:
I think jean peare is rigth, because at the end you are using that class in the execution, then you are testing the others class there.

Reply:
Remember that Validation.js is like an "extension" from Execution.js functionality refactored and encapsulated in another class to keep the code simple, so when we are testing the Execution by default the Validation test is involved, as Motopshyco mentioned. That's the reason Validation class doesn't need another unit test, and it would be redundant.